test/Elastica/Aggregation/SignificantTermsTest.php

```diff
@@ -15,17 +15,17 @@ protected function _getIndexForTest()
         $colors = ['blue', 'blue', 'red', 'red', 'green', 'yellow', 'white', 'cyan', 'magenta'];
         $temperatures = [1500, 1500, 1500, 1500, 2500, 3500, 4500, 5500, 6500, 7500, 7500, 8500, 9500];
 
-        $mapping = new Mapping($index->getType('test'), [
+        $mapping = new Mapping($index->getType('_doc'), [
             'color' => ['type' => 'keyword'],
             'temperature' => ['type' => 'keyword'],
         ]);
-        $index->getType('test')->setMapping($mapping);
+        $index->getType('_doc')->setMapping($mapping);
```

Comment:
I'm wondering if we should just do a separate PR to do all the `_doc` conversion. This should already work against 6.x and would make this PR much smaller.

Reply:
done #1574 :)